code/modules/events/spacevine.dm

```diff
@@ -231,13 +254,13 @@
 		var/datum/gas_mixture/gas_mix = turf.air
 		if(!gas_mix.gases[/datum/gas/nitrogen])
 			return
-		gas_mix.gases[/datum/gas/nitrogen][MOLES] = max(gas_mix.gases[/datum/gas/nitrogen][MOLES] - severity * holder.energy, 0)
+		gas_mix.gases[/datum/gas/nitrogen][MOLES] = max(gas_mix.gases[/datum/gas/nitrogen][MOLES] - GAS_MUTATION_REMOVAL_MULTIPLIER * holder.energy, 0)
```

Comment:
Instead of making mutation severities into defines you keep severeties as magic numbers in their typedefs but instead put copies of said severity values as defines when adressed in their type procs
Why not put defines into typedefs instead and keep the local var reliance inside procs instead?

Reply:
Or make both into typedefined vars for that matter since they are now separate entities

Reply:
there is no reason to have defines and and the values defined separately in typedef